pkg/logql/engine.go

```diff
@@ -426,7 +426,10 @@ func (q *query) evalVector(_ context.Context, expr *syntax.VectorExpr) (promql_p
 	}
 
 	if GetRangeType(q.params) == InstantType {
-		return s, nil
+		return promql.Vector{promql.Sample{
+			Point:  promql.Point{T: q.params.Start().UnixNano() / int64(time.Millisecond), V: value},
```

Comment:
Instead of `t.UnixNano()/int64(time.Millisecond)` we should use `t.UnixMilli()`

From the docs:

> // UnixMilli returns t as a Unix time, the number of milliseconds elapsed since
> // January 1, 1970 UTC. The result is undefined if the Unix time in
> // milliseconds cannot be represented by an int64 (a date more than 292 million
> // years before or after 1970). The result does not depend on the
> // location associated with t.

Reply:
done.thanks👍